torch/fx/graph_module.py

```diff
@@ -627,7 +627,7 @@ def wrapped_call(self, *args, **kwargs):
                 if "eval_with_key" in topmost_framesummary.filename:
                     print(generate_error_message(topmost_framesummary),
                           file=sys.stderr)
-                raise e.with_traceback(None)
+                raise e
```

Comment:
Discussed offline--consider changing to:
```python
    raise e.with_traceback(None)
else:
    raise e
```

Reply:
addressed, thanks!